references/detection/train_pytorch.py

```diff
@@ -237,11 +241,14 @@ def main(args):
         img_folder=os.path.join(args.train_path, 'images'),
         label_path=os.path.join(args.train_path, 'labels.json'),
         img_transforms=Compose([
-            T.Resize((args.input_size, args.input_size)),
             # Augmentations
             T.RandomApply(T.ColorInversion(), .1),
             ColorJitter(brightness=0.3, contrast=0.3, saturation=0.3, hue=0.02),
         ]),
```

Comment:
if the initial behaviour is still correct for straight boxes, I suggest making this conditional as well (only changing it if args.rotation)

If the initial behaviour was wrong (that rotation should be performed after photometric augmentations), then your design is correct

Reply:
No it is right in that case, so what should we do ? do a Compose .... if rotation else Compose .... ?

Reply:
Or you want to do double resize when rotation is on ?

Reply:
I think we can only answer this by experimenting. But until then, by default, let's use the same as before (resize in img transforms when there is no rotation)

We certainly don't want a double resize :sweat_smile: